Package.swift

```diff
@@ -66,6 +66,10 @@ let package = Package(
       name: "FirebaseFunctions",
       targets: ["FirebaseFunctions"]
     ),
+    .library(
+      name: "FirebaseInAppMessaging-Beta",
```

Comment:
Let's chat a bit more about this offline - I'm concerned about the transition when it goes GA that it'll be a big break requiring migration, and we may have to keep that `beta` around until the next breaking change.

Reply:
One option is documenting how to change in the Build Phase UI

Reply:
Will flush out the mechanics of the beta->GA flow in a subsequent task and doc.

Reply:
<img width="852" alt="Screen Shot 2020-08-12 at 9 34 22 AM" src="https://user-images.githubusercontent.com/73870/90042509-a37c2180-dc7f-11ea-8ee4-9871218a5178.png">